src/olympia/amo/tests/test_amo_utils.py

```diff
@@ -97,7 +97,8 @@ def test_resize_transparency_for_P_mode_bug_1181221():
     ('en_US', 'en-US'),
     ('en_us', 'en-US'),
     ('FR', 'fr'),
-    ('el', 'el')
+    ('el', 'el'),
+    ('x_zh_cn', 'x-ZH'),  # see mozilla/addons-server#3375
```

Comment:
Could you make these links to the issues instead? It's possible someone might not know what this means... but even if they did it's easy to have something I can click/copy+paste.